src/Qir/Execution/Tools/Utility/Clock.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Quantum.Qir.Utility
+{
+    public class Clock : IClock
```

Comment:
It is currently leveraged by the controller which gets access to this class by referencing the `Microsoft.Quantum.Qir.Tools` package. Since making this public is not ideal, I'll remove this class as well as the `IClock` interafce and use just `DateTimeOffset`.

Reply:
Removed.